### PR TITLE
full waypoint updater

### PR DESCRIPTION
**Pull request by thejerrytan**

### CODE REVIEW

File: ros/src/waypoint_updater/waypoint_updater.py [hunk 1]
```diff
@@ -23,14 +23,15 @@
 '''
 
 LOOKAHEAD_WPS = 200 # Number of waypoints we will publish. You can change this number
-
+MAX_DECEL = 10
```
Comment (ser94mor) [inline ros/src/waypoint_updater/waypoint_updater.py]: MAX_DECEL in another module (`waypoint_loader.py`) is set to `1.0`.

File: ros/src/waypoint_updater/waypoint_updater.py [hunk 2]
```diff
@@ -74,11 +75,40 @@ def get_closest_waypoint_idx(self):
...
+
+    def generate_lane(self):
         lane = Lane()
-        lane.header = self.base_waypoints.header
-        lane.waypoints = self.base_waypoints.waypoints[closest_idx:closest_idx + LOOKAHEAD_WPS]
-        rospy.loginfo("closest_idx %s " % closest_idx)
-        self.final_waypoints_pub.publish(lane)
+        closest_idx = self.get_closest_waypoint_idx()
+        farthest_idx = closest_idx + LOOKAHEAD_WPS
+        base_waypoints = self.base_lane.waypoints[closest_idx:farthest_idx]
```
Comment (ser94mor) [inline ros/src/waypoint_updater/waypoint_updater.py]: `self.base_lane` is not specified in `__init__()`. Here use `self.base_waypoints` instead.

Comment (thejerrytan) [reply ros/src/waypoint_updater/waypoint_updater.py]: nice catch! thanks

File: ros/src/waypoint_updater/waypoint_updater.py (hunk 1 above)
Comment (thejerrytan) [reply ros/src/waypoint_updater/waypoint_updater.py]: alright, this is the right value to use as corroborated by dbw.launch file in twist controller package

```
<?xml version="1.0"?>
<launch>
    <node pkg="twist_controller" type="dbw_node.py" name="dbw_node">
        <param name="vehicle_mass" value="1736.35" />
        <param name="fuel_capacity" value="13.5" />
        <param name="brake_deadband" value=".1" />
        <param name="decel_limit" value="-1." />
        <param name="accel_limit" value="1." />
        <param name="wheel_radius" value="0.2413" />
        <param name="wheel_base" value="2.8498" />
        <param name="steer_ratio" value="14.8" />
        <param name="max_lat_accel" value="3." />
        <param name="max_steer_angle" value="8." />
    </node>
</launch>

```

**Pull request merged**